cluster/up.sh

```diff
@@ -18,7 +18,9 @@ set -ex
 
 source ./cluster/kubevirtci.sh
 CNAO_VERSIOV=0.35.0
-KUBEVIRT_VERSION=v0.29.0
+KUBEVIRT_FIXED_MAJOR =v0.31
```

Comment:
Name this KUBEVIRT_MAJOR_MINOR or KUBEVIRT_VERSION since FIXED does not means anything here.

Reply:
ack you're right